picoquic/picoquic_internal.h

```diff
@@ -805,7 +813,8 @@ uint32_t picoquic_create_packet_header(
     picoquic_connection_id_t * srce_cnxid,
     uint8_t* bytes,
     uint32_t * pn_offset,
-    uint32_t * pn_length);
+    uint32_t * pn_length,
+    int contains_retrans);
 
```

Comment:
I don't think that this is right. The fact that a packet "contains a retransmission" is an artifact of the current code organization. Also, in a multi-path environment, a loss on one path may be corrected by a retransmission on another path -- and that can also happen in a migration scenario.

Bottom line, no, we should not add a "contains retransmission" parameter in the "create packet header" call.